fluentd/include/Gemfile.lock

```diff
@@ -10,47 +10,51 @@ GEM
     addressable (2.8.0)
       public_suffix (>= 2.0.2, < 5.0)
     aws-eventstream (1.1.1)
-    aws-partitions (1.481.0)
-    aws-sdk-core (3.118.0)
+    aws-partitions (1.483.0)
+    aws-sdk-cloudwatchlogs (1.43.0)
+      aws-sdk-core (~> 3, >= 3.119.0)
+      aws-sigv4 (~> 1.1)
+    aws-sdk-core (3.119.0)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.45.0)
-      aws-sdk-core (~> 3, >= 3.118.0)
+    aws-sdk-kms (1.46.0)
+      aws-sdk-core (~> 3, >= 3.119.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.97.0)
-      aws-sdk-core (~> 3, >= 3.118.0)
+    aws-sdk-s3 (1.98.0)
+      aws-sdk-core (~> 3, >= 3.119.0)
       aws-sdk-kms (~> 1)
       aws-sigv4 (~> 1.1)
-    aws-sdk-sqs (1.41.0)
-      aws-sdk-core (~> 3, >= 3.118.0)
+    aws-sdk-sqs (1.42.0)
+      aws-sdk-core (~> 3, >= 3.119.0)
       aws-sigv4 (~> 1.1)
     aws-sigv4 (1.2.4)
       aws-eventstream (~> 1, >= 1.0.2)
     concurrent-ruby (1.1.9)
     cool.io (1.7.1)
     domain_name (0.5.20190701)
       unf (>= 0.0.5, < 1.0.0)
-    elasticsearch (7.13.3)
-      elasticsearch-api (= 7.13.3)
-      elasticsearch-transport (= 7.13.3)
-    elasticsearch-api (7.13.3)
+    elasticsearch (7.14.0)
```

Comment:
This seems to have produced an issue for the `fluent-plugin-elasticsearch` plugin

Reply:
I see, so it's not just gem elasticsearch 7.14.0, it's the combination of that + the plugin. That makes sense.